### PR TITLE
ci: Run metrics_sdk tests

### PR DESCRIPTION
**Pull request by kaylareopelle**
While reviewing some of the recent PRs related to metrics, I noticed the metrics_sdk specs aren't running on the ci.

This will not pass until #1532 is merged.

### CODE REVIEW

**Comment by arielvalentin** [comment]
@kaylareopelle looks like builds are failing 🫂

**Comment by kaylareopelle** [comment]
@arielvalentin - Yes! We have to wait until #1532 is merged before they'll pass. 😄

**Comment by kaylareopelle** [comment]
This should be working, now that #1532 has been merged. Can someone with access re-run the tests?

**Review by robbkidd — APPROVED** [review]
👍🏻 once #1550 and #1532 merge

one after the other, after the other

**Pull request merged**
